src/map2d/pcd2track.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+class TrackMaker:
+    """
+    todo:
+        - detect empty pcd sector?
+        - how to connect pcd's?
+
+    Class to convert point cloud data to track data.
+    From 2d batch of points we are calculating 2 matrices:
+    - horizontal gradient
+    - vertical gradient
+    Then setting magnitude of gradient as a square root of sum of squares of both gradients.
+    Based on that magnitude we are setting threshold to get track data,
+    where łazik can drive on.
+    0 - can't drive
+    1 - can drive
+    """
+    def __init__(self):
+        self.visualize = False # todo: implement visualization
+        self.gradient_magnitude = None
+
+        if self.visualize:
+            self.vis = "todo: implement visualization"
+    
+    def __del__(self):
+        if self.visualize:
+            self.vis.destroy_window()
+
+    def calculate_gradient_magnitude(self, pcd: np.array):
```

Comment:
to musi być typu open3d.cuda.pybind.geometry.PointCloud. z biblioteki open3d 
aby przejść z o3d do np.array  `points = np.asarray(pcd.points)`